bookmarks.html

```diff
@@ -10,29 +10,26 @@
     <header>
       <h1>Quiz App</h1>
     </header>
-    <!-- I will use question-card as a class later to style it - What is better to use - or _ when I write a class?  -->
     <section class="question-card">
-      <div class="div_question-card">
+      <div class="question-card__button">
         <h2>What is Riesling?</h2>
         <button aria-label="Klick here to get answere">Show Answer</button>
         <p>A Grape variety, that was esay</p>
         <button>Hide Answer</button>
       </div>
-      <!-- I will use categories as a class to style it later -->
-      <ul class="question-card_categories">
-        <li class="question-card_categories_list">#wine</li>
-        <li class="question-card_categories_list">#basic</li>
-        <li class="question-card_categories_list">#beverage</li>
+      <ul class="question-card__categories">
+        <li class="question-card__categories--list">#wine</li>
+        <li class="question-card__categories--list">#basic</li>
+        <li class="question-card__categories--list">#beverage</li>
```

Comment:
```suggestion
      <ul class="question-card__list">
        <li class="question-card__list-item">#wine</li>
        <li class="question-card__list-item">#basic</li>
        <li class="question-card__list-item">#beverage</li>
```
I'd probably prefer something like this. `--list` is a little unusual, because a list wouldn't be a state/modifier.
If you take a gander at how we're handling the `nav` component, you can see why I'd suggest that we stick to the structure we've established so far with `list` and `list-item`.

Something like

```
     <ul class="question-card__catergory-list">
        <li class="question-card__category-list-item">#wine</li>
        <li class="question-card__category-list-item">#basic</li>
        <li class="question-card__category-list-item">#beverage</li>
```
also works. This may feel a little unintuitive, because it almost feels like list should be its own block, but we need to remember that `question-card` is our component, aka the block.

Reply:
I will stick the to question-card__list & question-card__list-item